frontend/src/types.ts

```diff
@@ -3077,7 +3077,6 @@ export interface Survey {
     response_sampling_limit?: number | null
     response_sampling_daily_limits?: string[] | null
     enable_partial_responses?: boolean | null
-    is_publicly_shareable?: boolean | null
```

Comment:
we need a PR to remove this from the DB right? before merging this
added here: https://github.com/PostHog/posthog/pull/33994/files